data/src/error_analysis/README.md

```diff
@@ -0,0 +1,45 @@
+# Error Analysis tool for G2P
+
+This tool could be used for performing fine-grain error analysis of a G2P model. It reports a performance matrix of a test output generated by a G2P model. 
+
+Under this error analysis process, each test record generated from a G2P model is evaluated under two criterions. 
+a) if the hypothesized form matches the corpus prediction or not.
```

Comment:
Use Markdown lists instead of a), etc.: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#lists

Reply:
fixed.